community/faas/examples/network.yaml

```diff
@@ -0,0 +1,30 @@
+# Example on how to use the network and subnetwork templates
+#
+# Variables:
+#   RAND: Just a random string used mostly by the testing suite.
```

Comment:
Do we actually need to use RAND for tests? i think we can reuse the same network name each time. Examples would be cleaner without it.

Reply:
The reason for this is when two or more people need to fire the same tests against the same project or org. This not only helps firing tests in parallel, but later the reuse of testcases and fixtures for when building more complex/stacked tests.
This also helps quite a bit on the speed of development, since we don't need to wait for a deployment to be deleted in order to run the same test again.
My 2c is to keep this only for the great flexibility/speed it gives us, but think of a way to make this prettier later on. What do you think?

Reply:
I see those reasons, but ultimately think the downside of having a more confusing example outweighs the benefits.

Regarding firing tests in parallel, different people should be doing it on different projects. I'd assume you'd delete the existing deployment before rerunning anyways.

I would like to see this fixed for this PR.